src/components/NewsLetterSignUp.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+
+export default function NewsLetterSignUp() {
+  const [buttonColour, setButtonColour] = useState(false);
+
+  const HandleButtonColour = () => {
+    setButtonColour(!buttonColour);
+  };
```

Comment:
This is extremely unclear what this really does. when you setButtonColour, the expected behavior is that I'm setting a color to change to but this is a boolean. when it's a boolean the naming should reflect the binary nature of the action.

Reply:
changed to changeColourButton

Reply:
That's still not good enough unfortunately. changeColourButton implies that I can change the color. 

generally when it comes to booleans, you would want a naming convension of "is___" to denote a true or false value.

Reply:
resolved.